src/diffusers/pipelines/flux/pipeline_flux_prior_redux.py

```diff
@@ -356,13 +409,29 @@ def __call__(
             returning a tuple, the first element is a list with the generated images.
         """
 
+        # 1. Check inputs. Raise error if not correct
+        self.check_inputs(
+            prompt,
+            prompt_2,
+            image,
+            prompt_embeds=prompt_embeds,
+            pooled_prompt_embeds=pooled_prompt_embeds,
+        )
```

Comment:
`image` should be first and missing `prompt_embeds_scale`/`pooled_prompt_embeds_scale`